server/bin/pbench-audit-server.sh

```diff
@@ -711,6 +718,23 @@ function verify_archive {
     return $cnt
 }
 
+function verify_reception {
```

Comment:
No sure this code belongs in the audit server, since the reception area is very dynamic, with multiple actors adding files remotely, and the "shim" moving them to the archive area in a non-atomic way.

Can you review the shim code and see where we could add this kind of report there?

Reply:
@portante Sure